core/iwasm/aot/arch/aot_reloc_xtensa.c

```diff
@@ -257,9 +257,11 @@ apply_relocation(AOTModule *module, uint8 *target_section_addr,
 
             /* check relative offset boundary */
             if (relative_offset < -256 * BH_KB || relative_offset > -4) {
-                set_error_buf(error_buf, error_buf_size,
-                              "AOT module load failed: "
-                              "target address out of range.");
+                set_error_buf(
+                    error_buf, error_buf_size,
+                    "AOT module load failed: "
+                    "target address out of range.\n"
+                    "Try using `wamrc --size-level=0` to generate .literal island");
```

Comment:
Line is too long, code guideline CI ran failed.

Reply:
sorry, update again